cclib/parser/qchemparser.py

```diff
@@ -691,20 +691,26 @@ def extract(self, inputfile, line):
                 self.append_attribute('time', float(tokens[8]))
 
             # Extract the atomic numbers and coordinates of the atoms.
-            if 'Standard Nuclear Orientation (Angstroms)' in line:
-                if not hasattr(self, 'atomcoords'):
-                    self.atomcoords = []
+            if 'Standard Nuclear Orientation' in line:
+                if "Angstroms" in line:
+                    def convertor(x):
```

Comment:
How about `convertor = lambda x: x` and `convertor = utils.convertot(...)` below?

Reply:
Next time I'll listen to my gut rather than the linter.